lib/main_containers.pm

```diff
@@ -186,7 +186,11 @@ sub load_container_tests {
     foreach (split(',\s*', $runtime)) {
         my $run_args = OpenQA::Test::RunArgs->new();
         $run_args->{runtime} = $_;
-        if (is_container_image_test()) {
+
+        if ($runtime eq 'kubectl') {
```

Comment:
Are you sure this makes sense? I moved it to the beginning, because I though that choosing the runtime tests have precedence over images tests on a runtime.

Reply:
yes, it's a container host, and therefore it should be in the same place as the container hosts :)

Reply:
The preference is set by these conditions and according variables. That's why we have that block for host tests, so let's try to put everything together.

Reply:
Ahhh, now I see what you mean. Yes that makes sense. Done.